chia/daemon/keychain_proxy.py

```diff
@@ -108,9 +108,11 @@ def handle_error(self, response: WsRpcMessage):
                 message = error_details.get("message", "")
                 raise MalformedKeychainRequest(message)
             else:
-                err = f"{response['data'].get('command')} failed with error: {error}"
-                self.log.error(f"{err}")
-                raise Exception(f"{err}")
```

Comment:
This is where the original `'NoneType' object is not subscriptable` exception was being raised.

Reply:
`.get()` returns `None` if the key isn't present.  Unrelated to this situation, you can specify an alternate default if you want.

```pycon
Python 3.10.1 (main, Jan 12 2022, 20:28:45) [GCC 9.3.0] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> {}.get("something")
>>> print({}.get("something"))
None
```

So `response` must be `None` actually.  But... Above there is `response["data"]` so it can't be `None`.

How as this identified as the source?

Reply:
Sorry about that, not sure why my memory of the root cause pointed at this area. You're correct that this isn't the main issue. I think during the course of making changes I noticed that 'None' was referenced as the command that failed, so perhaps that's where the confusion arose.